ZacharyBohn/url-shortener
# Fast api

# Fast API endpoints and DB

- Endpoints are now defined, connected to services, and dynamo db
- Tests with moto used to mock aws dynamo db
- standardized formatting for the project with vs code project settings

## TODO

- [x] Fix running the project. When running the project with `fastapi dev main.py`, the imports fail. I'm using absolute imports. I have another branch that switched to relative, but that failed to run the tests at all. Seems like there is hacky ways to get around this, and that Python was designed poorly. >.<
- [x] Write unit tests for utilities

## How to test

- Run `python test.py` in the root directory. All tests should pass.
- Run `fastapi dev main.py` to ensure that the project runs.
